Examples align (#75)
ArekBalysNordic/connectedhomeip: examples/all-clusters-minimal-app/nrfconnect/README.md

The example is based on
[Matter](https://github.com/project-chip/connectedhomeip) and Nordic

> Semiconductor's nRF Connect SDK, and works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power 802.15.4 Thread or Wi-Fi network. 

Reviewer: ```suggestion
Semiconductor's nRF Connect SDK, and was created to facilitate testing and
certification of a Matter device communicating over a low-power, 802.15.4 network, either Thread or Wi-Fi.
```

> Support for both Thread and Wi-Fi is mutually exclusive and depends on the hardware platform, so only one protocol can be supported for a specific lock device. In the case of Thread, this device works as a Thread Sleepy End Device. You can use this sample as a reference for creating your application.

Reviewer: ```suggestion
The example behaves as a Matter accessory, that is a device that can be paired
into an existing Matter network and can be controlled by this network. 

In the case of Thread, this device works as a Thread Sleepy End Device.
Support for both Thread and Wi-Fi is mutually exclusive and depends on the hardware platform, so only one protocol can be supported for a specific lock device.
```

<hr>

## IPv6 network support

The development kits for this sample offer the following IPv6 network support for Matter:

* Matter over Thread is supported for ``nrf52840dk_nrf52840``, ``nrf5340dk_nrf5340_cpuapp``, and ``nrf21540dk_nrf52840``.

> * Matter over Wi-Fi is supported for ``nrf5340dk_nrf5340_cpuapp`` with the ``nrf7002_ek`` shield attached or for ``nrf7002dk_nrf5340_cpuapp``.

Reviewer: Move this section under Requirements, on the same level as Supported devices (###). Also update the table of contents.

[nRF Connect platform overview](../../../docs/guides/nrfconnect_platform_overview.md)
to read more about the platform structure and dependencies.

> By default, the Matter accessory device has IPv6 networking disabled. You must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or Wi-Fi network. You have to make the device discoverable manually (for security reasons). See [Bluetooth LE advertising](#bluetooth-le-advertising) to learn how to do it. The controller must get the commissioning information from the Matter accessory device and provision the device into the network.

Reviewer: ```suggestion
By default, the Matter accessory device has IPv6 networking disabled. You must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or Wi-Fi network. You have to make the device discoverable manually (for security reasons). See [Bluetooth LE advertising](#bluetooth-le-advertising) to learn how to do this. The controller must get the commissioning information from the Matter accessory device and provision the device into the network.
```

[nRF Connect platform overview](../../../docs/guides/nrfconnect_platform_overview.md)
to read more about the platform structure and dependencies.

The sample uses buttons for changing the lock and device states, and LEDs to show the state of these changes. You can test it in the following ways:

- Standalone, using a single DK that runs the door lock application.

- Remotely over the Thread or the Wi-Fi protocol, which in either case requires more devices, including a Matter controller that you can configure either on a PC or a mobile device.

> By default, the Matter accessory device has IPv6 networking disabled. You must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or Wi-Fi network. You have to make the device discoverable manually (for security reasons). The controller must get the commissioning information from the Matter accessory device and provision the device into the network.

Reviewer: Move this whole line to line 58, before the list about standalone/remote testing.

> By default, the Matter accessory device has IPv6 networking disabled. You must pair it with the Matter controller over Bluetooth® LE to get the configuration from the controller to use the device within a Thread or Wi-Fi network. You have to make the device discoverable manually (for security reasons). The controller must get the commissioning information from the Matter accessory device and provision the device into the network.

Reviewer: Also about this line: we are adding information about manual discovery, but there is no mention of this requirement in the Testing section or elsewhere in the document, and so even the UI instructions are confusing. I'd suggest adding the relevant information to the Testing section.

Author: You can find it in the `Bluetooth LE advertising` section. Maybe it's worth to add a link to that section in this sentence.

To start the rendezvous, the controller must get the commissioning information
from the Matter device. The data payload is encoded within a QR code, printed to
the UART console, and shared using an NFC tag. NFC tag emulation starts
automatically when Bluetooth LE advertising is started and stays enabled until
Bluetooth LE advertising timeout expires.

> #### hread or Wi-Fi provisioning

Reviewer: Add a blank line and fix typo:
```suggestion

#### Thread or Wi-Fi provisioning
```

Last part of the rendezvous procedure, the provisioning operation involves

> sending the Thread / Wi-Fi network credentials from the Matter controller to the Matter device. As a result, the device joins the Thread or Wi-Fi network and can

Reviewer: ```suggestion
sending the Thread or Wi-Fi network credentials from the Matter controller to the Matter device. As a result, the device joins the Thread or Wi-Fi network and can
```

| [nRF52840 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK)         | `nrf52840dk_nrf52840`      | <details><summary>nRF52840 DK</summary><img src="../../platform/nrfconnect/doc/images/nRF52840_DK_info-medium.jpg" alt="nRF52840 DK"/></details>        |
| [nRF5340 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF5340-DK)           | `nrf5340dk_nrf5340_cpuapp` | <details><summary>nRF5340 DK</summary><img src="../../platform/nrfconnect/doc/images/nRF5340_DK_info-medium.jpg" alt="nRF5340 DK"/></details>           |
| [nRF52840 Dongle](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-Dongle) | `nrf52840dongle_nrf52840`  | <details><summary>nRF52840 Dongle</summary><img src="../../platform/nrfconnect/doc/images/nRF52840_Dongle-medium.jpg" alt="nRF52840 Dongle"/></details> |

> | [nRF7002 DK](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf7002.html#nrf7002dk-nrf5340)   | `nrf7002dk_nrf5340_cpuapp` | <details><summary>nRF7002DK</summary><img src="../../platform/nrfconnect/doc/images/nrf7002dk.jpg" alt="nRF7002 DK"/></details>    |

Reviewer: I don't see any mention of the ``nrf7002_ek`` or ``nrf21540dk_nrf52840`` here, yet you mention them in the IPv6 network support section...

    Bluetooth LE.

-   _Short Flash Off (950ms on/50ms off)_ &mdash; The device is fully

>     provisioned but does not yet have full Thread network or service

Reviewer: ```suggestion
    provisioned, but does not yet have full Thread network or service
```

    connectivity.

-   _Solid On_ &mdash; The device is fully provisioned.

**Button 1** can be used for the following purposes:

-   _Pressed for less than 3 s_ &mdash; Initiates the OTA software update

>     process. This feature is disabled by default but can be enabled by

Reviewer: ```suggestion
    process. This feature is disabled by default, but can be enabled by
```

    following the
    [Building with Device Firmware Upgrade support](#building-with-device-firmware-upgrade-support)
    instructions.

> -   _Pressed for more than 3 s_ &mdash; initiates the factory reset of the     device. 

Reviewer: ```suggestion
-   _Pressed for more than 3 s_ &mdash; initiates the factory reset of the device. 
```
